src/usr/local/buildpack/tools/flux.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+tool_path=$(find_tool_path)
+if [[ -z "${tool_path}" ]]; then
+  if [ -z $TOOL_VERSION ]; then
+    TOOL_VERSION=$(curl https://api.github.com/repos/fluxcd/flux2/releases/latest -sL | grep tag_name | sed -E 's/.*"([^"]+)".*/\1/' | cut -c 2-)
+  fi
+  INSTALL_DIR=$(get_install_dir)
+  base_path=${INSTALL_DIR}/${TOOL_NAME}
+  tool_path=${base_path}/${TOOL_VERSION}
+  mkdir -p ${tool_path}
+
+  ARCH=amd64
+  BIN_URL="https://github.com/fluxcd/flux2/releases/download/v${TOOL_VERSION}/flux_${TOOL_VERSION}_linux_${ARCH}.tar.gz"
+  curl -sL ${BIN_URL} -o /tmp/flux.tar.gz
+  mkdir -p /tmp/flux
+  tar -C /tmp/flux/ -zxvf /tmp/flux.tar.gz
```

Comment:
You should have write rights to `${tool_path}`
```suggestion
  tar -C ${tool_path} -zxvf /tmp/flux.tar.gz
```

is there only the `flux` cli inside the gzip file? if not, can we only extract the cli?

Reply:
Yep, there's just a single binary in the `tar.gz` file.